legend-engine-pure-code-compiled-core/src/main/resources/core/external/language/java/generation/expressionGeneration.pure

```diff
@@ -229,7 +229,7 @@ function <<access.private>> meta::external::language::java::transform::processFu
 function <<access.private>> meta::external::language::java::transform::processConcreteFunctionExpression(fe: FunctionExpression[1], conventions: Conventions[1], debug: meta::pure::tools::DebugContext[1]):Code[1]
 {
    print(if($debug.debug,|$debug.space+'processConcreteFunctionExpression '+$fe.func->elementToPath()+'\n',|''));
-         
+   let newCon = ^$conventions(visitedFunctions += $fe->cast(@FunctionExpression));
```

Comment:
Can we move this inside the if that starts on line 246?  Seems we dont use this outside the if.